optionals/compat_rhs_afrf3/CfgWeapons.hpp

```diff
@@ -332,4 +332,12 @@ class CfgWeapons {
         modes[] = {};
         picture = "\rhsafrf\addons\rhs_heavyweapons\data\ico\rhs_Kornet_9M133_2_msv_ca.paa";
     };
+    
+    class rhs_uniform_flora;
+    class rhs_uniform_df15: rhs_uniform_flora {
+        ACE_GForceCoef = 0.8;
+    };
+    class rhs_uniform_df15_tan: rhs_uniform_df15 {
+        ACE_GForceCoef = 0.8;
```

Comment:
because it inherits we don't need the config, but it doesn't really hurt

Reply:
I'd say remove it. It is just a reskin. The less external config, the better.